docs/topics/proc_authenticating-http-requests-using-the-web-interface.adoc

```diff
@@ -48,7 +48,7 @@ endif::node-js-sso-temp-workaround[]
 --
 NOTE: You can see the access (bearer) token in the command line output at the bottom of the page.
 
-.Authenticated {name-mission-secured} Booster Web Interface (as Alice)
+.Authenticated {name-example-secured} Example Web Interface (as Alice)
```

Comment:
Is this user name with a capital A ("Alice")? Or is it "alice"?

Reply:
The name of the user is Alice but her user name is 'alice'.